README.md

```diff
@@ -85,7 +85,7 @@ cobra-cli add <new-command>
 ### Available Commands
 
 | Command                    | Description                                                                         |
-|----------------------------|-------------------------------------------------------------------------------------|
+| -------------------------- | ----------------------------------------------------------------------------------- |
```

Comment:
What's this change for? Or was it from vscode autoformatting?

Reply:
Yeah this was just auto formatting. Not sure why it changed...